src/modules/Dashboard/pages/Applet/AppletMultiInformant.tsx

```diff
@@ -35,37 +53,85 @@ export const AppletMultiInformant = () => {
-                </StyledFlexTopCenter>
-              </Tooltip>
-            )}
-          </StyledFlexTopCenter>
-          <LinkedTabs hiddenHeader={hiddenHeader} tabs={appletTabs} isCentered={false} />
+          {!hiddenHeader && (
+            <StyledFlexSpaceBetween
+              gap={theme.spacing(1.6)}
+              margin={`${theme.spacing(1.2)} ${theme.spacing(3.4)}`}
+            >
+              <StyledFlexTopCenter gap={theme.spacing(1.6)}>
+                {!!appletData.image && <StyledAppletLogo src={appletData.image} />}
+                <StyledHeadlineLarge>{appletData.displayName}</StyledHeadlineLarge>
+                {!!appletData?.description && (
+                  <Tooltip
+                    title={appletData.description as string}
+                    disableFocusListener
+                    disableTouchListener
+                    arrow
+                    placement="right-end"
+                  >
+                    <StyledFlexTopCenter sx={{ cursor: 'pointer', paddingX: theme.spacing(0.8) }}>
+                      <Svg id="more-info-outlined" fill={palette.outline_variant} />
+                    </StyledFlexTopCenter>
+                  </Tooltip>
+                )}
+              </StyledFlexTopCenter>
+
+              <StyledFlexTopBaseline gap={theme.spacing(1)}>
+                <Button
+                  sx={{
+                    gap: theme.spacing(1),
+                    color: variables.palette.on_surface_variant,
+                  }}
+                  onClick={() => {
+                    setIsExportOpen(true);
+                    Mixpanel.track('Export Data click');
+                  }}
+                >
+                  <Svg id="export" width={24} height={24} />
+                  <StyledTitleMedium as="span" sx={{ color: variables.palette.on_surface_variant }}>
+                    {t('export')}
+                  </StyledTitleMedium>
+                </Button>
+                <Link
+                  data-testid="dashboard-tab-settings"
+                  to={generatePath(page.appletSettings, {
+                    appletId,
+                  })}
+                >
+                  <Svg id="settings" fill={isSettingsSelected ? variables.palette.primary : ''} />
+                </Link>
+              </StyledFlexTopBaseline>
+            </StyledFlexSpaceBetween>
+          )}
+
+          <LinkedTabs
+            animateTabIndicator={false}
+            defaultToFirstTab={false}
+            hiddenHeader={hiddenHeader}
+            isCentered={false}
+            tabs={appletTabs}
+          />
+
+          {isSettingsSelected && (
+            <StyledPanel hiddenHeader={hiddenHeader}>
+              <DashboardAppletSettings />
+            </StyledPanel>
+          )}
         </>
       )}
+      <ExportDataSetting
+        isExportSettingsOpen={isExportOpen}
+        onExportSettingsClose={() => {
+          setIsExportOpen(false);
+        }}
+      />
```

Comment:
This component uses the applet data as the source of data to export, but because it doesn't accept it as a prop, rendering it before the appletData is available leads to weird behavior if the user selects "All time" as the date range

https://github.com/ChildMindInstitute/mindlogger-admin/assets/14842108/ac546a55-14c0-4429-8384-c514fe968263

That said, using "All time" seems to be borked anyway on the BE, but that's a different issue

```suggestion
      {appletData && (
        <ExportDataSetting
          isExportSettingsOpen={isExportOpen}
          onExportSettingsClose={() => {
            setIsExportOpen(false);
          }}
        />
      )}
```

Reply:
Nice catch!